src/index.js

```diff
@@ -107,78 +107,81 @@ const App = () => {
                         })
                     }
                 }
-                
-                if (colToSort === 'task'){
+
+                if (colToSort === 'task') {
                     todos.sort((a, b) => {
-                        if (a.name.toLowerCase() > b.name.toLowerCase()){
+                        if (a.name.toLowerCase() > b.name.toLowerCase()) {
                             return direction === 'asc' ? -1 : 1;
-                        } 
-                        if (a.name.toLowerCase() < b.name.toLowerCase()){
+                        }
+                        if (a.name.toLowerCase() < b.name.toLowerCase()) {
                             return direction === 'asc' ? 1 : -1;
-                        } 
+                        }
                     });
                 }
-            }  
+            }
         }
         return todos;
     }
 
-    const sortedTodos = sortTodos(); 
+    const sortedTodos = sortTodos();
 
     return (
-        <div> 
+        <div>
             <h1>A To Do List</h1>
             <table>
                 <thead>
                     <tr>
                         <th>
                             Completion
                         </th>
-                        <th> 
-                            <button type="button" onClick = {() => onSortColumn('task')}>
-                                Task 
+                        <th>
+                            <button type="button" onClick={() => onSortColumn('task')}>
+                                Task
                                 <i className="icon">{preSortIcon}</i>
                             </button>
                         </th>
-                        
+
                         <th>
-                            <button type="button" onClick = {() => onSortColumn('date')}>
+                            <button type="button" onClick={() => onSortColumn('date')}>
                                 Due Date
                                 <i className="icon">{preSortIcon}</i>
                             </button>
                         </th>
                     </tr>
-                </thead>  
+                </thead>
 
-                <tbody>  
+                <tbody>
                     <ToDoItems todos={sortedTodos} onCompleteItem={handleCompleteItem} />
-                </tbody>  
+                </tbody>
             </table>
 
-            <input 
-                className={!todoInputValue && hasTaskInputBeenTouched ? 'invalid' : null}
-                type='text' 
+            <input
+                className={!todoInputValue && hasTaskInputBeenTouched ? 'invalid' : ''}
```

Comment:
actually I dont think there is a class needed for the false here

Reply:
You can consider using `classnames` library to perform condition classNames in the future.